src/main/resources/application.yaml

```diff
@@ -288,6 +288,18 @@ notification:
       representative:
         emailId: 97c58e23-c11f-40b3-b981-2d4cfa38b8fd
         smsId: bb3df0ea-8259-43c4-95de-9eef96206575
+    question_round_issued:
+      emailId: ccfbfcac-847d-4e55-955c-096802c131a0
+    follow_up_question_round_issued:
+      emailId: 71efa426-bfa9-473e-b3cf-da5d99e39461
+    question_deadline_elapsed:
+      emailId: 48f2b7d1-cb71-4f60-b145-dfcf63cc4fcd
+    question_deadline_reminder:
+      emailId: 29820a6a-1aa4-4518-8d08-f06f09c5d808
+    continuous_online_hearing_relisted:
+      emailId: 46cc347a-1a5c-40ce-993a-49769dfce626
```

Comment:
How have these ended back in? Look like they are related to COR functionality which has been removed.

Reply:
Have removed these entries